.github/workflows/create-gui-pr.yml

```diff
@@ -38,15 +34,15 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "18.13"
+          node-version: '18.13'
```

Comment:
Reading from the .nvmrc file would be ideal, but I don’t think there is an easy way to do that from within the actions. Not so easy: check-out the repo, find the file, and annoyingly parse it (I believe it supports more formats than what we use) and use it here.

Without that, I would still rather keep this value in the code base because then you only need to change it in the code base and not _also_ in the repo settings.